config/locales/en/prior_authority.yml

```diff
@@ -70,18 +70,15 @@ en:
           - On %{requested} we asked you to update your application before we could complete the assessment. This was due by %{deadline}.
           - As this update was not made by the required date this application has been automatically closed.
           - If you make a new application you should include the requested update.
+        sent_back_explanations_html:
+          - Updates are needed to proceed with your application.
+          - Review the requests and resubmit your application by %{deadline}.
+          - After this date your application will be automatically closed.
+        update_application: Update application
```

Comment:
**nit** since it is all html anyway i had done this as below. which feels simpler since it removes the loop from the view

```
        sent_back:
          ....
          laa_response_blurb_html: |
            <p>Updates are needed to proceed with your application.</p>
            <p>Review the requests and resubmit your application by <strong>%{resubmission_deadline}</strong>.</p>
            <p>After this date your application will be automatically closed.</p>
```

Reply:
I'm not really a fan of having any markup in yaml files - without syntax highlighting it makes it even easier to not spot invalid HTML.